Signal/src/ViewControllers/ThreadSettings/OWSConversationSettingsViewController.m

```diff
@@ -185,6 +185,17 @@ - (BOOL)isGroupThread
     return [self.thread isKindOfClass:[TSGroupThread class]];
 }
 
+-(BOOL)isPrivateGroupChat
```

Comment:
This happens to be used in conjunction with `isGroupThread` so it works, but this method doesn't actually return whether the given thread is a private group chat or not. It returns whether the thread in question isn't an RSS feed or public chat.

Reply:
Better would be:

```
if (self.isGroupThread) {
    TSGroupThread *thread = (TSGroupThread *)self.thread;
    return !thread.isRSSFeed && !thread.isPublicChat;
}
return false;
```